examples/sample_measurement/measurement.py

```diff
@@ -56,6 +62,7 @@ class Color(Enum):
 @measurement_service.output("Bool out", nims.DataType.Boolean)
 @measurement_service.output("String out", nims.DataType.String)
 @measurement_service.output("Enum out", nims.DataType.Enum, enum_type=Color)
+@measurement_service.output("Double2DArray out", nims.DataType.Double2DArray)
```

Comment:
The client generator doesn't support message outputs other than DoubleXYData yet: https://github.com/ni/measurement-plugin-python/blob/main/packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py#L156

Reply:
Added Support for output for Double2DArray in Client Generator in newer commits.